jetstream/outcomes/firefox_desktop/newtab_do_no_harm.toml

```diff
@@ -0,0 +1,27 @@
+friendly_name = "New Tab - Do No Harm"
+description = """
+Guardrail metrics for the New Tab. Includes user experience and revenue-related metrics for Search, Pocket and AdMarketplace Sponsored Tiles.
+"""
+
+[metrics]
+weekly = [
+    "newtab_searches", 
```

Comment:
@scholtzan , is this the right way to refer to this [metric](https://github.com/mozilla/metric-hub/blob/rebecca-burwei-patch-10/definitions/firefox_desktop.toml#L863)?

Reply:
it is

Reply:
thanks :)